llvm/test/CodeGen/WebAssembly/conv.ll

```diff
@@ -1,3 +1,4 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 4
```

Comment:
Assertions in this test and the below (`simd-sext-inreg.ll`) are not autogenerated. Why are these said to be autogenerated?